terraform/gcp/security/gcp-compute-disk-encryption.yaml

```diff
@@ -0,0 +1,27 @@
+rules:
+- id: gcp-compute-disk-encryption
+  patterns:
+    - pattern: |
+        resource "google_compute_disk" $ANYTHING {
+          ...
+        }
+    - pattern-not-inside: |
+        resource "google_compute_disk" $ANYTHING {
+          ...
+          disk_encryption_key {
+            raw_key = ...

```

Comment:
Same question as above; commenting here for reference later

Reply:
According to [google_compute_instance](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_instance) terraform docs `kms_key_self_link` as well.